tests/tests/tier0/bluechi-change-port-with-p-cmd-option/main.fmf

```diff
@@ -0,0 +1,3 @@
+summary: "Test if bluechi agent/controller accepts changing port
+by passing command line argument `-p`/`--port`"
+id:
```

Comment:
In the `tests` directory run `tmt test id` to assign this new test an ID

Reply:
thx, done